app/models/concerns/with_assignments.rb

```diff
@@ -19,7 +19,6 @@ def has_messages_for?(user)
   end
 
   # TODO: When the organization is used in this one, please change guide.pending_exercises
```

Comment:
This second TODO shall be addressed in another PR!

Reply:
Great!

Reply:
Awesome!